### DISCUSSION TITLE
Best way to play music/soundboard?

### DISCUSSION THREAD

**Discussion by mansman12** [streamdeck-linux-gui/streamdeck-linux-gui#119] [Q&A]
Using Windows with the Stream Deck app, you could play sounds natively or use a plugin (I used SoundDeck) to play music/music playlist.

What would be the best way of doing both of these on the streamdeck-linux-gui? Any suggestions welcome!

**Comment by Quintar** [comment]
mplayer in a script, maybe?

**Reply by mansman12** [reply]
Thanks, I ended up using mpv with playerctl to control playlists.

**Comment by pkoerner81929** [comment]
https://flathub.org/apps/io.github.Soundux seems to be an option. They support various hot keys that could be launched from Streamdeck.

**Reply by mansman12** [reply]
Thanks, currently been looking at Soundux for the soundboard side. Waiting for them to do their update as they are currently in the middle of a rewrite.
